### ISSUE TITLE
Line chart clustering does not work in Bokeh

### ISSUE THREAD

**Issue by markwatsonatx** [ibm-cds-labs/pixiedust#201]
As a user, I want to cluster my line charts in Bokeh

### Expected behavior

I am able to cluster my line charts in Bokeh

### Actual behavior

Get the following error message: Warning: 'Cluster By' ignored when you have multiple Value Fields but subplots options selected

### Steps to reproduce the behavior

Use Boston Crime data
df = pixiedust.sampleData(7)
display(df)
Choose line chart
Set keys = district
Set values = type
Choose bookeh
Set Cluster by to month

**Comment by DTAIEB** [comment]
Fixes in PR #230

**Comment by markwatsonatx** [comment]
I am now seeing a new issue following the same repro steps above. Here is the stacktrace:

```
2017-03-06 16:36:54,924 - pixiedust.display.display.Display - DEBUG - values: ['month']
2017-03-06 16:36:54,924 - pixiedust.display.display.Display - DEBUG - Calling with month
2017-03-06 16:36:54,924 - pixiedust.utils.template - DEBUG - Template already qualified pixiedust.display.chart.renderers.baseChartDisplay:baseChartOptionsDialogBody.html
2017-03-06 16:36:54,950 - pixiedust.display.chart.renderers.baseChartDisplay - DEBUG - Found cache data for E748E514A418452CAB229524CABCBAD9. Validating integrity...
2017-03-06 16:36:54,951 - pixiedust.display.chart.renderers.baseChartDisplay - DEBUG - Cache data not validated for key extraFields. Expected Value is ['X']. Got ['month']. Destroying it!...
2017-03-06 16:36:55,939 - pixiedust.display.display.Display - ERROR - Unexpected error while trying to render BaseChartDisplay
Traceback (most recent call last):
  File "/pixiedust/pixiedust/display/chart/renderers/baseChartDisplay.py", line 392, in doRender
    chartFigure = self.doRenderChart()
  File "/pixiedust/pixiedust/display/chart/renderers/bokeh/bokehBaseDisplay.py", line 72, in doRenderChart
    charts = self.createBokehChart()
  File "/pixiedust/pixiedust/display/chart/renderers/bokeh/lineChartDisplay.py", line 90, in createBokehChart
    fig.line(x = pivot.index.values, y = pivot[str(col)].values, color = Colors.hexRGB( 1.*i/2 ), legend=col if self.showLegend() else None)
  File "/opt/conda/lib/python2.7/site-packages/pandas/core/frame.py", line 2059, in __getitem__
    return self._getitem_column(key)
  File "/opt/conda/lib/python2.7/site-packages/pandas/core/frame.py", line 2066, in _getitem_column
    return self._get_item_cache(key)
  File "/opt/conda/lib/python2.7/site-packages/pandas/core/generic.py", line 1386, in _get_item_cache
    values = self._data.get(item)
  File "/opt/conda/lib/python2.7/site-packages/pandas/core/internals.py", line 3543, in get
    loc = self.items.get_loc(item)
  File "/opt/conda/lib/python2.7/site-packages/pandas/indexes/base.py", line 2136, in get_loc
    return self._engine.get_loc(self._maybe_cast_indexer(key))
  File "pandas/index.pyx", line 132, in pandas.index.IndexEngine.get_loc (pandas/index.c:4433)
  File "pandas/index.pyx", line 156, in pandas.index.IndexEngine.get_loc (pandas/index.c:4363)
KeyError: '8'
```

**Comment by DTAIEB** [comment]
Issue was related to the column type being numerical. 
Fixed in PR #236
